### PR TITLE
Document solution to handling conflicting filename

### LINKED ISSUE #1080: Documentation Missing for Existing File Upload Conflict Resolution Option
#### Category
- [ ] Question
- [x] Documentation issue
- [ ] Bug

#### Background

[Here](https://docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content?view=odsp-graph-online) is the relevant endpoint. Currently, if you upload a file with the same filename as an existing file, the existing file is overwritten. To change this behavior so that filename conflicts are handled by renaming the new file, you can append the following query parameter: `@name.conflictBehavior=rename`. This is currently not documented anywhere

#### Expected or Desired Behavior

The optional request parameter for handling filename conflicts for file uploads should be recorded in the documentation for the endpoint.

#### Observed Behavior

Files uploaded with the same name as that of an existing file overwrite the existing file.

#### Steps to Reproduce

1. Upload any file using the endpoint linked above.
2. Upload a different file with the same name as that of the file in part 1.
3. Observe that the file from part 1 was overwritten with the file from part 2.
4. Upload the file from part 1 again, this time adding the query parameter `@name.conflictBehavior=rename`.
5. Observe that both files exist in the drive with the last one to be uploaded having been renamed.

### PR DESCRIPTION
**Pull request by alexwaibelmsft**
The default behavior of this endpoint is to overwrite any existing file with the same filename as that of the file you are attempting to upload. This can be resolved with an optional query parameter, however this was previously undocumented.

This PR resolves #1080

### CODE REVIEW

**Comment by daspek** [comment]
Docs Build status updates of commit _[cad6a30](https://github.com/alexwaibelmsft/onedrive-api-docs/pull/1072/commits/cad6a30413f5fb9a33d9ba973ef878d37aa8ce73)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_put_content.md](https://github.com/alexwaibelmsft/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_put_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content?view=odsp-graph-online&branch=pr-en-us-1072) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C5%5C30%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201905301633214252-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=6ToCzjy%2B1tj9sGBYyYPO2mIDnMtUmXjwAJXzJ3jqT0w%3D&st=2019-05-30T16%3A29%3A48Z&se=2019-06-30T16%3A34%3A48Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: docs/rest-api/api/driveitem_put_content.md [hunk 1]
```diff
@@ -118,6 +118,8 @@ Content-Type: application/json
 
 ## Remarks
 
+### Forbidden File Types
```
Comment (chackman) [inline docs/rest-api/api/driveitem_put_content.md]: Please state "Blocked File Types"

File: docs/rest-api/api/driveitem_put_content.md [hunk 2]
```diff
@@ -132,6 +134,13 @@ for more information on blocked file extensions.
 
 [blocked-extensions]: https://support.office.com/en-us/article/Types-of-files-that-cannot-be-added-to-a-list-or-library-30be234d-e551-4c2a-8de8-f8546ffbf5b3?ui=en-US&rs=en-AU&ad=AU&fromAR=1#__toc355959797]
 
+### Handling Filename Conflicts
+
+The default behavior for this endpoint is to overwrite existing documents which have the same filename as that of the newly uploaded file. In order to avoid this, you can specify that the new file should be renamed in the case of a conflict by using one of the following query parameters:
```
Comment (chackman) [inline docs/rest-api/api/driveitem_put_content.md]: Could you please make the wording more like driveitem.md, like so:

By default, this API will replace an existing item with the same name.  To customize the conflict resolution behavior, add a query parameter named *@microsoft.graph.conflictBehavior* with values *fail*, *replace*, or *rename*, to the request.

Comment (chackman) [reply docs/rest-api/api/driveitem_put_content.md]: And please name the section "Conflict Resolution Behavior"

File: docs/rest-api/api/driveitem_put_content.md (hunk 1 above)
Comment (alexwaibelmsft) [reply docs/rest-api/api/driveitem_put_content.md]: Fixed

File: docs/rest-api/api/driveitem_put_content.md (hunk 2 above)
Comment (alexwaibelmsft) [reply docs/rest-api/api/driveitem_put_content.md]: Both of these changes have been made.

**Comment by daspek** [comment]
Docs Build status updates of commit _[30c4ebe](https://github.com/alexwaibelmsft/onedrive-api-docs/pull/1072/commits/30c4ebee2a81270a62a9ce15dad41ac92d012a16)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_put_content.md](https://github.com/alexwaibelmsft/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_put_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content?view=odsp-graph-online&branch=pr-en-us-1072) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C14%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201906141600240981-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=07BV18F8bjLExIogcTPdHaArgKB4OAK1wKZJ4biUPyo%3D&st=2019-06-14T15%3A56%3A49Z&se=2019-07-15T16%3A01%3A49Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by daspek** [comment]
Docs Build status updates of commit _[d0cd7f2](https://github.com/alexwaibelmsft/onedrive-api-docs/pull/1072/commits/d0cd7f2f6fc9b31b3a753691869c07bbb72b71b5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_put_content.md](https://github.com/alexwaibelmsft/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_put_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content?view=odsp-graph-online&branch=pr-en-us-1072) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C17%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201906171544526979-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=tXcS9edQXoD%2FWZG1V7ikWYbNKHtn%2BcFsXlXXMrR3Vns%3D&st=2019-06-17T15%3A40%3A53Z&se=2019-07-18T15%3A45%3A53Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by daspek** [comment]
Docs Build status updates of commit _[0069103](https://github.com/alexwaibelmsft/onedrive-api-docs/pull/1072/commits/006910325f634c2d7ede7a9ea03ee2cedbe5b58b)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[docs/rest-api/api/driveitem_put_content.md](https://github.com/alexwaibelmsft/onedrive-api-docs/blob/patch-1/docs/rest-api/api/driveitem_put_content.md) | :white_check_mark:Succeeded | [View](https://review.docs.microsoft.com/en-us/onedrive/developer/rest-api/api/driveitem_put_content?view=odsp-graph-online&branch=pr-en-us-1072) (>= odsp-graph-online) |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2019%5C6%5C17%5C1b9326b4-eb89-99a2-12be-09cf82b65094%5CPullRequest%5C201906171548104391-1072%5Cworkflow_report.html?sv=2016-05-31&sr=b&sig=lN2PCtGf4NALRxQdbF1oP0%2FXoKdJPjMvkYjfxmit0Sg%3D&st=2019-06-17T15%3A44%3A22Z&se=2019-07-18T15%3A49%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by alexwaibelmsft** [comment]
@chackman can you merge the branch in when you have a chance? I don't have repo access. I've made the italics change you requested.

**Pull request merged**